planetiler-core/src/test/java/com/onthegomap/planetiler/archive/TileArchiveConfigTest.java

```diff
@@ -33,4 +42,51 @@ void testPmtiles() {
     assertEquals(TileArchiveConfig.Format.PMTILES,
       TileArchiveConfig.from("file:///output.mbtiles?format=pmtiles").format());
   }
+
+  @ParameterizedTest
+  @EnumSource(TileArchiveConfig.Format.class)
+  void testByFormatParam(TileArchiveConfig.Format format) {
+    final var config = TileArchiveConfig.from("output?format=" + format.id());
+    assertEquals(format, config.format());
+    assertEquals(TileArchiveConfig.Scheme.FILE, config.scheme());
+    assertEquals(Path.of("output").toAbsolutePath(), config.getLocalPath());
+    assertEquals(Map.of("format", format.id()), config.options());
+  }
+
+  @ParameterizedTest
+  @EnumSource(TileArchiveConfig.Format.class)
+  void testGetPathForMultiThreadedWriter(TileArchiveConfig.Format format) {
+    final var config = TileArchiveConfig.from("output?format=" + format.id());
+    if (!format.supportsConcurrentWrites()) {
+      assertThrows(UnsupportedOperationException.class, () -> config.getPathForMultiThreadedWriter(0));
+      assertThrows(UnsupportedOperationException.class, () -> config.getPathForMultiThreadedWriter(1));
+    } else {
+      assertEquals(config.getLocalPath(), config.getPathForMultiThreadedWriter(0));
+      final Path p = config.getPathForMultiThreadedWriter(1);
+      switch (format) {
+        case FILES -> assertEquals(p, config.getLocalPath());
+        default -> assertEquals(config.getLocalPath().getParent().resolve(Paths.get("output1")), p);
+      }
+    }
+  }
+
+  @Test
+  void testExistsForFilesArchive(@TempDir Path tempDir) throws IOException {
+    final Path out = tempDir.resolve("outdir");
+    final var config = TileArchiveConfig.from(out + "?format=files");
```

Comment:
Currently when `format=` is missing `TileArchiveConfig` defaults to mbtiles. I wonder if when the output path has no extension or ends with a slash if it should default to `files` as a small ease of use improvement?

Reply:
done https://github.com/onthegomap/planetiler/pull/761/commits/1faf56611a11f9e92ec02f631262689c8a13e417